src/auth.rs

```diff
@@ -61,7 +61,8 @@ impl Authenticator {
         let bytes = format!("n,,{}", message).into_bytes();
         let binary = Binary(Generic, bytes);
 
-        let start_doc = doc! {
+        let start_doc =
+            doc! {
             "saslStart" => 1,
```

Comment:
Looks like rustfmt wants to move the macro invocation to a new line but then doesn't indent everything else. I'll fix that here and elsewhere and push a new commit.